frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.tsx

```diff
@@ -97,148 +287,69 @@ function StartingJobModal(props: StartingJobModalProps) {
-        <br />
-      </React.Fragment>
-    ) : null;
+  let initialLayerName = layers.length === 1 ? layers[0].name : null;
+  let initialOutputSegmentationLayerName = getReadableNameOfVolumeLayer(layers[0], tracing);
+  if (fixedSelectedLayer) {
+    initialLayerName = fixedSelectedLayer.name;
+    initialOutputSegmentationLayerName = getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing);
   }
-
-  const renderUserBoundingBox = (bbox: UserBoundingBox | null | undefined) => {
-    if (!bbox) {
-      return null;
-    }
-
-    const upscaledColor = bbox.color.map((colorPart) => colorPart * 255) as any as Vector3;
-    const colorAsHexString = rgbToHex(upscaledColor);
-    return (
-      <>
-        <div
-          className="color-display-wrapper"
-          style={{
-            backgroundColor: colorAsHexString,
-            marginTop: -2,
-            marginRight: 6,
-          }}
-        />
-        {bbox.name} ({computeArrayFromBoundingBox(bbox.boundingBox).join(", ")})
-      </>
-    );
-  };
-
-  function BoundingBoxSelection(): React.ReactNode {
-    return isBoundingBoxConfigurable ? (
-      <React.Fragment>
-        <p>
-          Please select the bounding box for which the inferral should be computed. Note that large
-          bounding boxes can take very long. You can create a new bounding box for the desired
-          volume with the bounding box tool in the toolbar at the top. The created bounding boxes
-          will be listed below.
-        </p>
-        <div
-          style={{
-            textAlign: "center",
-          }}
-        >
-          <Select
-            showSearch
-            style={{
-              width: 400,
-            }}
-            placeholder="Select a bounding box"
-            optionFilterProp="children"
-            // @ts-expect-error ts-migrate(2322) FIXME: Type 'Element | null' is not assignable to type 'n... Remove this comment to see the full error message
-            value={renderUserBoundingBox(selectedBoundingBox)}
-            onChange={onChangeBoundingBox}
-            filterOption={(input, option) =>
-              // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-              option.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
-            }
-          >
-            {userBoundingBoxes.map((userBB) => (
-              <Select.Option key={userBB.id} value={userBB.id}>
-                {renderUserBoundingBox(userBB)}
-              </Select.Option>
-            ))}
-          </Select>
-        </div>
-        <br />
-      </React.Fragment>
-    ) : null;
-  }
-
-  const hasUnselectedOptions =
-    selectedColorLayerName == null || (isBoundingBoxConfigurable && selectedBoundingBox == null);
+  initialOutputSegmentationLayerName = `${
+    initialOutputSegmentationLayerName || "segmentation"
+  }_corrected`;
+  // TODO: Other jobs also have an output segmentation layer. The names for these jobs should also be configurable.
```

Comment:
This TODO needs to be kept, as the new `OutputSegmentationLayerNameFormItem` should also be used for the inferral jobs. I did not do it in this pr to avoid making this pr even bigger.

Reply:
please open a follow-up issue for this :)

Reply:
issue: #6197